sdcm/nemesis.py

```diff
@@ -687,6 +798,141 @@ def _modify_table_property(self, name, val, filter_out_table_with_counter=False,
         with self.tester.cql_connection_patient(self.target_node) as session:
             session.execute(cmd)
 
+    def _get_all_non_compact_tables(self, allowed=None):
```

Comment:
What is the allow argument to this function ? is is a filter of some kind ? some docstring might help to clearify it